installer/darwin/create_dmg.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# Create a read-only disk image of the contents of a folder
+# This script is derived from the create-dmg utility
+# https://github.com/create-dmg/create-dmg
+
+# Bail out on any unhandled errors
+set -ex;
+
+VOLUME_NAME=$1
+APP_BUNDLE_PATH=$2
+DMG_PATH=$3
+
+VOLUME_ICON_FILE=installer/darwin/invest.icns
+BACKGROUND_FILE=installer/darwin/background.png
+EULA=LICENSE.txt
+
+SRC_FOLDER=$(cd $APP_BUNDLE_PATH > /dev/null; pwd)
+DMG_DIRNAME=$(dirname $DMG_PATH)
+DMG_DIR=$(cd $DMG_DIRNAME > /dev/null; pwd)
+DMG_NAME=$(basename $DMG_PATH)
+DMG_TEMP_NAME=$DMG_DIR/tmp.$DMG_NAME
+
+# remove .DS_Store from source folder, if there is one
+rm $SRC_FOLDER/.DS_Store || true
+
+# remove intermediate dmg from previous run, if there is one
+rm -f $DMG_TEMP_NAME || true
+
+# Use Megabytes since hdiutil fails with very large byte numbers
+function blocks_to_megabytes() {
+    # Add 1 extra MB, since there's no decimal retention here
+    MB_SIZE=$((($1 * 512 / 1000 / 1000) + 1))
+    echo $MB_SIZE
+}
+
+function get_size() {
+    # Get block size in disk
+    bytes_size=$(du -s "$1" | sed -e 's/    .*//g')
+    echo $(blocks_to_megabytes $bytes_size)
+}
+
+# Create the DMG with the hdiutil estimation
+hdiutil create \
+    -srcfolder $SRC_FOLDER \
+    -volname "$VOLUME_NAME" \
+    -fs HFS+ `# type of file system to write to the image`\
```

Comment:
Ah, I've been wondering about how to include comments like this for a long time.